### PR TITLE
Fix LittleFS sample after rebasing onto zephyr-v3.3.0

### PR DESCRIPTION
**Pull request by xakep-amatop**
After rebasing onto zephyr-v3.3.0 littlefs sample doesn't work with R-Car MMC driver.

The LittleFS sample may be used for both SDMMC and MMC disk drivers, so make it possible to work with different volume disk driver names.

Use mmc disk driver instead of sdmmc, because for littlefs sample we use mmc node with a sub-node disk compatible with 'zephyr,mmc-disk' driver.

### CODE REVIEW

**Comment by xakep-amatop** [comment]
note: I'm going to open pull-request with the first one commit from this pull to mainline

**Comment by xakep-amatop** [comment]
update documentation of the sample

**Review by firscity — APPROVED** [review]
`Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>`

File: samples/subsys/fs/littlefs/README.rst [hunk 1]
```diff
@@ -88,7 +88,7 @@ To build the test:
    :gen-args: -DCONF_FILE=prj_blk.conf
    :compact:
 
-One can also set ``CONFIG_SDMMC_VOLUME_NAME`` to provide the mount point name
+One can also set ``CONFIG_SDMMC_VOLUME_NAME`` or ``CONFIG_MMC_VOLUME_NAME`` (in case of MMC) to provide the mount point name
```
Comment (lorc) [inline samples/subsys/fs/littlefs/README.rst:91]: Isn't this line too long?

File: samples/subsys/fs/littlefs/src/main.c [hunk 2]
```diff
@@ -309,6 +309,15 @@ static int littlefs_mount(struct fs_mount_t *mp)
 #endif /* CONFIG_APP_LITTLEFS_STORAGE_FLASH */
 
 #ifdef CONFIG_APP_LITTLEFS_STORAGE_BLK_SDMMC
+
+#if defined(CONFIG_DISK_DRIVER_SDMMC)
+#define DISK_NAME CONFIG_SDMMC_VOLUME_NAME
+#elif IS_ENABLED(CONFIG_DISK_DRIVER_MMC)
+#define DISK_NAME CONFIG_MMC_VOLUME_NAME
+#else
+#error "No disk device defined, is your board supported?"
```
Comment (lorc) [inline samples/subsys/fs/littlefs/src/main.c:318]: What if both drivers are enabled? Or it is not possible?

Comment (xakep-amatop) [reply samples/subsys/fs/littlefs/src/main.c:318]: it is definitely possible that we have both drivers or even three (flash map) enabled
but the sample can't work with a few devices at the same time, so we should control what we want to test by a board config or a sample config

File: samples/subsys/fs/littlefs/README.rst (hunk 1 above)
Comment (xakep-amatop) [reply samples/subsys/fs/littlefs/README.rst:91]: as far as I see, we have even a longer line in this file, the line 57
and GitHub checkpatch hasn't shown any errors

**Comment by xakep-amatop** [comment]
added ack

**Comment by lorc** [comment]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by xakep-amatop** [comment]
added ack

**Pull request merged**